packages/adminrouter/extra/src/Makefile.common

```diff
@@ -68,14 +69,14 @@ devkit:
 				/bin/bash -c "\$$OPENRESTY_COMPILE_SCRIPT" && \
 		docker commit $$(docker ps -a -q -f name=$(DEVKIT_NAME)) \
 			mesosphere/$(DEVKIT_NAME):full && \
-		docker rm $(DEVKIT_NAME) && \
+		docker rm -f $(DEVKIT_NAME) && \
 		docker rmi -f mesosphere/$(DEVKIT_NAME):noresty; \
 	fi
 
 .PHONY: update-devkit
 update-devkit: clean-devkit-container
 	@# Let's suppress ALL the output:
-	@docker rm $(DEVKIT_NAME) &> /dev/null || true
+	@docker rm -f $(DEVKIT_NAME) &> /dev/null || true
```

Comment:
Is this not handled by `clean-devkit-container`?

Reply:
It is, I will remove it from here.